guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -69,7 +69,9 @@ To update the certificates on a currently running Katello installation, run:
 ----
 endif::[]
 
-. From the output of the `katello-certs-check` command, depending on your requirements, enter the `{foreman-installer}` command that deploys a certificate to a new {Project} or updates a certificate on a currently running {Project}.
+. From the output of the `katello-certs-check` command, depending on your requirements, enter the `{foreman-installer}` command that installs a new {Project} with custom SSL certificates or updates certificates on a currently running {Project}.
++
+If you are unsure what command to run, you can verify that {Project} is installed by checking if the file `/etc/foreman-installer/scenarios.d/.installed` exists. If the file exists, run the second `{foreman-installer}` command that updates certificates.
```

Comment:
```suggestion
If you are unsure which command to run, you can verify that {Project} is installed by checking if the file `/etc/foreman-installer/scenarios.d/.installed` exists. If the file exists, run the second `{foreman-installer}` command that updates certificates.
```

Reply:
For what it's worth, `.installed` for the Foreman scenario only started with 2.0. It was a katelloism prior to that.

Reply:
This is the trick that I've found out from @ehelms. Luckily this documentation is for Foreman 2.0+ only.